evaluation/numpy/segmentation_and_tracking_quality_test.py

```diff
@@ -0,0 +1,278 @@
+# coding=utf-8
+# Copyright 2021 The Deeplab2 Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Tests for segmentation_and_tracking_quality.py"""
+
+import unittest
+import numpy as np
+
+import segmentation_and_tracking_quality as stq
```

Comment:
from deeplab2.evaluation.numpy import segmentation_and_tracking_quality as stq

Reply:
Does that make sense if people want to use this in a stand-alone fashion? For example, I imagine people just copying this folder and it would be great, if it still works.

If yes, I am happy to change it.

Reply:
I see your point. The only issue is when we convert the code back to google codebase, `import segmentation_and_tracking_quality as stq` might create some dependency issues. But we should be able to add some rules to manually convert it.

Reply:
Maybe update the file-scope docstring in segmentation_and_tracking_quality.py saying that this file is stand-alone and the users could just copy the whole folder in their own use case.